.github/workflows/run_tests.yml

```diff
@@ -138,7 +138,7 @@ jobs:
         export ARM_TENANT_ID='${{ secrets.ACCEPTANCE_TESTS_ENV_TENANT_ID }}'
         export ARM_SUBSCRIPTION_ID='${{ secrets.ACCEPTANCE_TESTS_ENV_SUBSCRIPTION_ID }}'
         go clean -testcache
-        go test -p 10 -failfast -v ./... -run ^Test -coverprofile=test-coverage.out -timeout 300m
+        go test -p 10 -failfast -v ./... -run ^TestAccEnvironment -coverprofile=test-coverage.out -timeout 300m
```

Comment:
Don't we want to run all the tests?

Reply:
This is until the branch is ready for review